src/LibraryWindow.vala

```diff
@@ -643,11 +643,10 @@ public class Noise.LibraryWindow : LibraryWindowInterface, Gtk.Window {
 
         // hide playlists when media list is empty
         source_list_view.change_playlist_category_visibility (have_media);
+        statusbar.playlist_menubutton_sensitive = have_media;
```

Comment:
For completeness, should this not be:
```statusbar.playlist_menubutton_sensitive = folder_set && have_media;```

Just to make it the same as the check you're moving.